Uchu.World/Objects/GameObjects/Player.cs

```diff
@@ -57,6 +57,8 @@ internal Player()
             OnNotifyTamingBuildSuccessMessage = new Event<NotifyTamingBuildSuccessMessage>();
             OnLootPickup = new Event<Lot>();
             OnWorldLoad = new Event();
+            OnRailMovementReady = new Event();
```

Comment:
Coulnd't these be moved to the game object?

Reply:
Do you mean moving the event to the rail activator object, or having it still fire on the player but be defined in `GameObject.cs` instead of `Player.cs`?